main.css

```diff
@@ -0,0 +1,573 @@
+    color: #fff;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    padding-top: 50px;
+  }
+
+  .cards {
+    display: grid;
+    grid-template-columns: 1fr;
+    grid-template-rows: repeat(5, 1fr);
+    align-items: center;
+    justify-content: center;
+    gap: 10px;
+    width: 100%;
+    padding-top: 50px;
+    margin-left: 5px;
+  }
+
+  .card {
+    display: flex;
+    align-items: center;
+    gap: 30px;
+    background-color: #383b41;
+    width: 100%;
+    height: 100px;
+    padding: 6px 18px 22px;
+  }
+
+  .card:hover {
+    border: 2px solid #fff;
+  }
+
+  .card-img {
+    width: 65px;
+    height: 60px;
+  }
+
+  .card-title {
+    color: #ff5438;
+
+    /* font-size: 14.3px; */
+    font-weight: 700;
+    line-height: 25px;
+  }
+
+  .card-text {
+    height: 90%;
+    font-weight: normal;
+    width: 100%;
+  }
+
+  .join {
+    width: 75%;
+    height: 80px;
+    background-color: #ff4f38;
+    color: #fff;
+    margin-top: 30px;
+    font-weight: 900;
+    font-size: 20px;
+  }
+
+  .see-prog {
+    display: none;
+  }
+
+  .speakers-card {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+  }
+
+  .speakers {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+  }
+
+  .speaker {
+    display: flex;
+    flex-direction: row;
+    gap: 10px;
+    align-items: center;
+    margin-top: 20px;
+    margin-left: 30px;
+  }
+
+  .speaker-img {
+    width: 106px;
+    height: 106px;
+  }
+
+  .wrapper {
+    display: flex;
+    flex-direction: column;
+    gap: 10px;
+    margin-left: 10px;
+  }
+
+  .speaker-name {
+    font-size: 15px;
+    font-weight: 700;
+  }
+
+  .speaker-title {
+    color: #f78c89;
+    font-size: 10px;
+    font-weight: 700;
+  }
+
+  .speaker-desc {
+    font-size: 9px;
+    line-height: 17px;
+    font-weight: 700;
+  }
+
+  .hide-mobile {
+    display: none;
+  }
+}
+
+@media screen and (min-width: 768px) {
+  header {
+    height: 100%;
+    width: 100%;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    overflow: hidden;
+  }
+
+  .hidden {
+    display: none;
+  }
+
+  body::before {
+    content: '';
+    position: absolute;
+    top: 0;
+    left: 0;
+    width: 100%;
+    height: 100%;
+    opacity: 0.1;
+    z-index: -1;
+    background-image: url('./img/back-img1.png');
+    background-position: center;
+    background-repeat: no-repeat;
+    background-size: cover;
+
+    /* background-color: #fff; */
+    padding-top: 50px;
+    background-clip: border-box;
+  }
+
+  .list-1 {
+    display: flex;
+    background-color: #000b;
+    justify-content: end;
+    color: white;
+    padding-right: 250px;
+    height: 25px;
+  }
+
+  ul {
+    list-style: none;
+  }
+
+  .list-1 ul {
+    display: flex;
+    align-items: center;
+    gap: 30px;
+  }
+
+  a {
+    text-decoration: none;
+  }
+
+  li a {
+    color: #fff;
+  }
+
+  .mobile-menu {
+    display: none;
+  }
```

Comment:
- [ ] Please 🙏🏽 correct the navbar in the mobile version to be similar to the original design for a better UI.

| Original design | Your design |
|---|---|
| ![image](https://user-images.githubusercontent.com/58642949/234666658-a267badb-f021-4423-983a-d60699177ce0.png) | ![image](https://user-images.githubusercontent.com/58642949/234666749-750c9a70-e501-445c-84b7-bc155a3b4d72.png) |